polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -1018,8 +1018,8 @@ impl<T: Config> Pallet<T> {
 			let recipient_deposit = if system_channel { 0 } else { config.hrmp_recipient_deposit };
 
 			if request.confirmed {
-				if <paras::Pallet<T>>::is_valid_para(channel_id.sender) &&
-					<paras::Pallet<T>>::is_valid_para(channel_id.recipient)
+				if paras::Pallet::<T>::is_valid_para(channel_id.sender) &&
```

Comment:
If you ever get to explore https://github.com/trailofbits/dylint for our preferred syntax, I'd be very curious to hear about it :)